asserts/repair.go

```diff
@@ -162,25 +162,33 @@ func assembleRepair(assert assertionBase) (Assertion, error) {
 		return nil, err
 	}
 
-	// verify that modes is a list of only "run", "recover" and "install"
+	// validate that all base snap names are valid snap names
+	for _, b := range bases {
+		if err := naming.ValidateSnap(b); err != nil {
+			return nil, fmt.Errorf("cannot use %q as element in \"bases\": %v", b, err)
```

Comment:
invalid snap name %q in "bases" perhaps ?

Reply:
mmm that results in this for at least our current error unit test case:

```
assertion repair: invalid snap name \"foo....bar\" in \"bases\": invalid snap name: \"foo....bar\"
```

but maybe that's fine